.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -184,3 +188,38 @@ private object InlineHelp : WPComPluginBuild(
 	docsLink = "TODO",
 	withPRNotify = "false",
 )
+
+private object GutenbergUploadSourceMapsToSentry: BuildType() {
+	init {
+		name = "Upload Gutenberg Source Maps to Sentry";
+		id("WPComPlugins_GutenbergUploadSourceMapsToSentry");
+
+		params {
+			text(
+				name = "GUTENBERG_VERSION",
+				value = "",
+				label = "Gutenberg version",
+				description = "The Gutenberg version to upload source maps for (include the whole string, including the `v` prefix)",
+				allowEmpty = false
+			)
+		}
+
+		steps {
+			bashNodeScript {
+				name = "Upload source maps to Sentry"
+				scriptContent = """
+					# Install the Sentry CLI binary
+					curl -sL https://sentry.io/get-cli/ | bash
+
+					wget https://github.com/WordPress/gutenberg/releases/tag/%GUTENBERG_VERSION%/gutenberg.zip
+					unzip gutenberg.zip -d gutenberg
+					cd gutenberg
+
+					sentry-cli --auth-token %SENTRY_AUTH_TOKEN% releases --org a8c files "wpcom-test-01" delete --all
```

Comment:
What's the purpose of deleting the existing files?

Reply:
For now, we only have a single, fixed release, and I thought I'd start with a clean slate at every release, which will make it more manageable in the long run. The release is only used for GB at the moment, so that doesn't have any further consequences. I think, though, that we might as well not delete anything, since the paths will be different anyway for each upload (due to the version change), and if we use this release for other app's source-maps, then deleting here would be indeed a problem.

TL;DR: I thnk we can get rid of it.

Reply:
Cool! That's similar to what I was thinking, especially if we need to upload other apps

Reply:
Another benefit of not deleting is that we can have source-maps for multiple releases at the same time 🤔 which can be a good thing for enabling them on edge before we push to production.